plugins/texteditor.koplugin/main.lua

```diff
@@ -211,7 +211,7 @@ Do you want to proceeed?]]),
                 onConfirm = function(dir_path)
                     local file_input
                     file_input = InputDialog:new{
-                        title =  _("Enter new file filename"),
+                        title =  _("Enter filename"),
```

Comment:
New filename would also work if it's for a new file, but new file filename is stylistically awkward ;)

Reply:
Yes, new file filename sounded bad to me too :) but Enter filename sounds prettier to my ears now than Enter new filename.

Reply:
Thunar, for reference:
![screenshot_2018-08-07_20-50-40](https://user-images.githubusercontent.com/202757/43796110-92d4244e-9a83-11e8-9d81-815e64c5c1eb.png)